2026-T2/MLN/assignments/Assessment3/notebook/nb_cells_v5.py

```
@@ -0,0 +1,1213 @@
+"""v5 cells for MLN601 Assessment 3, in CRISP-DM order.
+
+Sections 1-3 (title, business, data understanding, preparation), section 4 (modelling),
+sections 5-6 (evaluation, lessons, appendices, references). Assembled by build_v5.py.
+"""
+
+md = lambda s: ("markdown", s.strip("\n"))
```

Ruff (0.16.1)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `md` as a `def`

(E731)